module/VuFind/src/VuFind/RecordTab/HierarchyTree.php

```diff
@@ -156,14 +156,9 @@ public function isFullHierarchyVisible()
                 return true;
             }
         }
-
         // Currently displaying top of tree?  Disable partial hierarchy:
-        if ($this->getActiveTree() == $recordDriver->getUniqueId()) {
-            return true;
-        }
-
         // Only if we got this far is it appropriate to use a partial hierarchy:
-        return false;
+        return $this->getActiveTree() == $recordDriver->getUniqueId();
```

Comment:
The comments above this line no longer make sense, since the logic has changed. Maybe it should be changed to something like:

```
// If displaying the top of the tree, we should show the full hierarchy;
// otherwise, if we got this far, it is appropriate to use a partial hierarchy.
```